index.js

```diff
@@ -49,28 +50,37 @@ client.on("message", function (message) {
     inventoryCommand(message);
   } else if (message.content.startsWith("!lookup")) {
     lookUpCommand(message);
+  } else if (message.content.startsWith("!remove")) {
+    removeCardCommand(message);
   }
 });
 
 const drawCommand = (message) => {
-  var ref = db.ref("users");
+  const ref = db.ref("users");
   ref.once("value", function (snapshot) {
     const userData = snapshot.val();
-    let userDraw = message.author.id;
-    let user = userData[userDraw];
+    const userID = message.author.id;
+    const user = userData[userID];
     const userDoesNotExist = !user;
     let timeDifference;
     if (user) {
       let lastBotUse = user.lastDrawTime;
       timeDifference = Date.now() - lastBotUse;
-      console.log(userData);
-      console.log(lastBotUse);
-      console.log(Date.now());
-      console.log(timeDifference);
+      const userInventory = userData[userID].inventory;
+      const inventorySize = Object.keys(userInventory).length;
+
+      if (inventorySize >= 60) {
```

Comment:
this is what's called a [Magic Number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants) -- a code smell where an arbitrary number that isnt -1, 0, 1 probably should be given a name. This has a couple advantages for you: if you set the deck size to a constant, you can easily change the deck size in 1 place rather than changing `60` here and `59` below.